lighthouse/host-path/beacon-deployment.yaml

```diff
@@ -46,6 +48,6 @@ spec:
           path: /data/lighthouse/beacon
       securityContext:
         runAsUser: 1001
-        runAsGroup: 2000
+        runAsGroup: 2000 
```

Comment:
There's an extra space. We can fix this format issue along with the security context value structure adjustment.